sway/config/output.c

```diff
@@ -397,6 +454,16 @@ bool apply_output_config(struct output_config *oc, struct sway_output *output) {
 
 	struct wlr_output *wlr_output = output->wlr_output;
 
+	if (output->current_config && compare_output_config(oc, output->current_config)) {
```

Comment:
This will lead to NULL pointer dereference if `oc` is NULL

Reply:
calling `apply_output_config(NULL, output)` is valid? I've been running this for a while without issue but I guess the rest of the code is defensive about that so might as well be too.